pt-BR/auth.json

```diff
@@ -1,6 +1,6 @@
 {
   "spec": {
-    "language": "pt-BR",
+    "language": "en",
```

Comment:
@marcosmartini it looks like the language is incorrectly changed to `en`

Reply:
@evgenyk Sorry, updated now